src/pages/AssetsList.vue

```diff
@@ -24,14 +24,14 @@
       <template #row="{ item }">
         <div class="assets-list-page__row">
           <BaseLink
-            :to="`/assets/${item.id}`"
+            :to="`/assets/${encodeURIComponent(item.id.definition.toString())}`"
```

Comment:
As it is a common use case both for `AssetId` and `AssetDefinitionId`, maybe worth adding a method like `toStringEscaped()` to both classes. (though only definition id needs it)